20_classes_lab/02.py

```diff
@@ -1,4 +1,17 @@
-"""
-Following code assumes a class named MyCounter
-Fill in the code so correct result is printed
-"""
+##### Exercise-2 ##################
+class MyCounter(object):
+    counter = []
+
+    def __init__(self):
+        self.counter.append(1)
+
+    @classmethod
+    def count(self):
+        return sum(MyCounter.counter)
+
+
+for _ in range(10):
+    c1 = MyCounter()
+
+# should print 10
+print(MyCounter.count())

```

Comment:
You can define it the same way you defined the counter list. However, inside the methods you need to prefix it with the name of the class as in `MyCounter.counter`.